### PR TITLE
WL-4677: Print issue on a Lessons page

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css [hunk 1]
```diff
@@ -337,10 +337,13 @@ a.edit-link:link, a.edit-link:visited, a.section-merge-link:link, a.section-merg
     -moz-border-radius: 7px;
     border-radius: 7px;
     border: 1px solid #222;
-    display:inline-block;
     vertical-align: top;
 }
 
+div.Mrphs-portalWrapper .column {
```
Comment (buckett) [inline lessonbuilder/tool/src/webapp/css/Simplepagetool.css:343]: A comment really helps explain why you're putting this property in it's own section.

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css [hunk 2]
```diff
@@ -1385,7 +1388,7 @@ div[role="dialog"] {
     background: url(../images/reload_h.gif) center left no-repeat
 }
 
-.right-col {
+div.Mrphs-portalWrapper .right-col {
```
Comment (buckett) [inline lessonbuilder/tool/src/webapp/css/Simplepagetool.css:1391]: I'd move this up to the other column stuff and again, comment about you're wanting to detect when it's in the portal and when it's not.

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css (hunk 1 above)
Comment (nicholaswilson100) [reply lessonbuilder/tool/src/webapp/css/Simplepagetool.css:343]: yup

**Comment by nicholaswilson100** [comment]
More code coming to a PR near you soon!

**Comment by buckett** [comment]
Doesn't merge at the moment.

**Pull request closed without merging**